src/components/Staking/StakingProductsCardGrid/StakingProductCard.tsx

```diff
@@ -191,7 +191,7 @@ export const StakingProductCard = ({
           {data.map(({ label, status }, idx) => (
             <li
               key={idx}
-              className={`my-4 me-0 ms-auto flex items-center gap-1 text-md leading-3 ${status === "false" && "text-body-medium"}`}
+              className={`my-4 me-0 ms-auto flex items-center gap-1 text-md leading-[1rem] ${status === "false" && "text-body-medium"}`}
```

Comment:
A fun shortcut with tailwind, you can append a `text-` sizing class with `/line-height-class`. We're using `text-md` here, which should really be `text-base`, which is 16px. You're assigning 1rem, which is a line height of 100%, or 1. This can be accomplished with tailwind class `leading-none`.

So to consolidate, you can do all this with `text-base/none`

```suggestion
              className={`my-4 me-0 ms-auto flex items-center gap-1 text-base/none ${status === "false" && "text-body-medium"}`}
```

Just thought I'd share! Going to commit that and pull in, thanks @Anish-Gupta1!